06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,264 @@
+const uniqueTermsBook = ['alice', 'falls', 'into', 'a', 'rabbit'];
+const books = require('../assets/books.json');
+const wrongdata = require('../assets/wrongdata.json');
+const emptyfile = require('../assets/emptyfile.json');
+const smallfile = require('../assets/smallfile.json');
+
```

Comment:
Thank you for pointing this out, I have rectified this issue, by renaming all my variables using the correct style guide.